src/Stryker.Core/Stryker.Core/Instrumentation/BaseEngine.cs

```diff
@@ -1,11 +1,10 @@
-﻿using System;
+using System;
 using Microsoft.CodeAnalysis;
 
 namespace Stryker.Core.Instrumentation
 {
     internal abstract class BaseEngine<T>: IInstrumentCode where T: SyntaxNode
     {
-
         protected BaseEngine(string markerId, string engineId)
         {
             InstrumentEngineID = engineId;
```

Comment:
I'd like to see we take the engineId from `GetType().Name`. That way all types get their own Id but we don't have to pass a string from each type to the base constructor. As we only create all engines once performance should not be impacted by this right?

Reply:
Good design.
done